package.json

```diff
@@ -1,7 +1,18 @@
 {
+  "private": true,
```

Comment:
we don't want to check-in this file.

please run: 
git checkout <branchname>
git checkout master -- package.json
git add .
git commit
git push -f origin HEAD

Reply:
Do the other changes and test before  to do this revert.